### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by CMUgarte**
Standard update. Just changed one Glottocode (Bookkeeping) in the language file.

### CODE REVIEW

File: cldf/languages.csv [hunk 1]
```diff
@@ -12,7 +12,7 @@ Thai,Thai,thai1261,Thai,tha,Eurasia,14.192,100.671,Tai-Kadai,Tai-Kadai/ Thai [th
 Korean,Korean,kore1280,Korean,kor,Eurasia,37.5,128.0,Koreanic,Koreanic/ Korean [kor],
 Dholuo,Dholuo,luok1236,Luo (Kenya and Tanzania),luo,Africa,-0.93296,34.4896,Nilo-Saharan,Nilo-Saharan/ Dholuo [luo],"BlountLUOGrammar.,Malandra1957English-LwooDictionary,Gorman1972ADholuo,Odhiambo2010ParlonsKenya"
 Hungarian,Hungarian,hung1274,Hungarian,hun,Eurasia,46.9068585714,19.6555271429,Uralic,Uralic/ Hungarian [hun],"Orszagh1990,Siptar2000"
-ChuanqiandianClusterMiao,Chuanqiandian Cluster Miao,chua1256,Chuanqiandian Cluster Miao,cqd,Eurasia,22.5638,101.95,Hmong-Mien,Hmong-Mien/ Chuanqiandian Cluster Miao [cqd],"XiongYuYou2014Miao,XingGui2004:Dialect,ShiLiYun2016366Miao"
+ChuanqiandianClusterMiao,Chuanqiandian Cluster Miao,firs1234,First Vernacular Hmong,cqd,,22.5638,101.95,Hmong-Mien,Hmong-Mien/ Chuanqiandian Cluster Miao [cqd],"XiongYuYou2014Miao,XingGui2004:Dialect,ShiLiYun2016366Miao"
```
Comment (FredericBlum) [inline cldf/languages.csv]: Feels unintuitive to have this mapped to a phylum. @chrzyki Do we have precedents for this?

**Review by FredericBlum — COMMENTED** [review]
Lets discuss the mapping of the language, seems weird to me to have it mapped to a phylum.

File: cldf/languages.csv (hunk 1 above)
Comment (chrzyki) [reply cldf/languages.csv]: Agreed, particularly in the context of this dataset. There are instances of that in other datasets, yes, but as far as I recall this is normally avoided.

Comment (FredericBlum) [reply cldf/languages.csv]: Then I'd propose to kick out this glottocode and leave it unmapped for now. After that we can merge

Comment (chrzyki) [reply cldf/languages.csv]: Agreed!

**Review by chrzyki — APPROVED** [review]
After changing the phylum mapping.

**Pull request merged**